crates/pop-contracts/tests/contract.rs

```diff
@@ -13,7 +13,11 @@ fn setup_test_environment() -> std::result::Result<tempfile::TempDir, Error> {
 	let temp_dir = tempfile::tempdir().expect("Could not create temp dir");
 	let temp_contract_dir = temp_dir.path().join("test_contract");
 	fs::create_dir(&temp_contract_dir)?;
-	create_smart_contract("test_contract", temp_contract_dir.as_path(), &Template::Standard)?;
+	create_smart_contract(
+		"test_contract",
+		temp_contract_dir.as_path(),
+		&ContractTemplate::Standard,
```

Comment:
Do we need a borrow here?

Reply:
Yes, `create_smart_contract` requires a borrowed template.